kratos/includes/constitutive_law.h

```diff
@@ -539,6 +542,22 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
      */
     virtual SizeType GetStrainSize();
 
+    /**
+     * @return The initial state of strains/stresses/F
+     */
+    void SetInitialState(InitialState& rInitialState)
+    {
+        mpInitialState = &rInitialState;
+    }
+
+    /**
+     * @return The initial state of strains/stresses/F
+     */
+    InitialState* GetpInitialState()
```

Comment:
```suggestion
    InitialState* pGetInitialState()
```

needs also a `const` version